### ISSUE TITLE
Recreating the results, Hyperparameters.

### ISSUE THREAD

**Issue by adibaig1** [priya-dwivedi/aerial_pedestrian_detection#17]
Hi, 
I am currently trying to train keras retinanet form scratch using stanford dataset but the mAP i get fluctuates in the range of 20-30, whereas you got a great one of 63. 

I was just wondering if you could share the hyperparameters you used to train your model? Did you change the steps per epoch? 

I noticed that the repository is using 0.25 alpha value of focal loss but wouldn't it be better to use 0.75?
